fixes.inc

```diff
@@ -5142,6 +5169,52 @@ static _FIXES_DetermineOS()
 	forward bool:operator>=(File:a, _:b);
 #endif
 
+/**
+ * <remarks>
+ * Declare the function to reset the car-jacked player id to invalid id.
+ * </remarks>
+ */
+ 
+#if FIX_SilentTeleport
+	forward FIXES_RemoveJackedId(playerid);
+	public FIXES_RemoveJackedId(playerid) return FIXES_gsJackedId[playerid] = INVALID_PLAYER_ID;
+#endif
+
+/**
+ * <fixes>SilentTeleport</fixes>
+ */
+#if defined _ALS_SetVehiclePos
+     #error _ALS_SetVehiclePos defined 
+#endif 
+native BAD_SetVehiclePos(vehicleid, Float:x, Float:y, Float:z) = SetVehiclePos; 
+
+/**
+ * <fixes>SilentTeleport</fixes>
+ */
+
+#if FIX_SilentTeleport
+     stock FIXES_SetVehiclePos(vehicleid, Float:x, Float:y, Float:z)
+     { 
+		 for(new i; i < MAX_PLAYERS; i++)
```

Comment:
You need to use _FIXES_FOREACH loop instead

Reply:
You should check discord i tagged u as well about this, regardless i've changed it as well.